### PR TITLE
[3] Pause sync sites job

### PR DESCRIPTION
**Pull request by JR-G**
## Context

Whilst we rename this table we need to temporarily pause the sync sites job

## Link to Trello card

https://trello.com/c/yzG11Oym/183-site-uuids-transition-2-2-remove-the-old-site-model-spike

## Things to check

- [x] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by JR-G** [comment]
[Site UUIDS: Transition [2/2] - Remove the old site model [SPIKE]](https://trello.com/c/yzG11Oym/183-site-uuids-transition-2-2-remove-the-old-site-model-spike)

File: config/clock.rb [hunk 1]
```diff
@@ -10,9 +10,9 @@ class Clock
 
   # More-than-hourly jobs
 
-  every(10.minutes, 'IncrementalSyncAllFromTeacherTrainingPublicAPI', skip_first_run: true) do
-    TeacherTrainingPublicAPI::SyncAllProvidersAndCoursesWorker.perform_async(true)
-  end
+  # every(10.minutes, 'IncrementalSyncAllFromTeacherTrainingPublicAPI', skip_first_run: true) do
```
Comment (cwrw) [inline config/clock.rb:13]: We'll need to silence `CourseSyncNotSucceededForAnHour
CourseSyncNotSucceededForAnHour` when this is merged, hopefully not for long, and then run the full sync after to catchup

**Review by cwrw — APPROVED** [review]
needs a rebase otherwise 👍

File: config/clock.rb (hunk 1 above)
Comment (JR-G) [reply config/clock.rb:13]: 👍

**Pull request merged**